cogs/ReactForRole.py

```diff
@@ -1078,9 +1110,19 @@ def create_tables(self):
         UNIQUE (guild_id, role_id)
         );
         """
+        sql_create_inline_all_status_table = """
+        CREATE TABLE IF NOT EXISTS InlineAllStatus (
+        guild_id integer NOT NULL,
```

Comment:
guild_id should be text, this is fine for now as I can change it later.

Reply:
Best to change now, removes the need for changing later

Reply:
Changed guild_id to be text